src/Apps/Order2/Routes/Checkout/Components/DeliveryOptionsStep/Order2DeliveryOptionsStep.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Flex, Text } from "@artsy/palette"
+import {
+  CheckoutStepName,
+  CheckoutStepState,
+} from "Apps/Order2/Routes/Checkout/CheckoutContext/types"
+import { useCheckoutContext } from "Apps/Order2/Routes/Checkout/Hooks/useCheckoutContext"
+
+interface Order2DeliveryOptionsStepProps {}
+
+export const Order2DeliveryOptionsStep: React.FC<
+  Order2DeliveryOptionsStepProps
+> = () => {
+  const { steps } = useCheckoutContext()!
+
+  const stepState = steps?.find(
+    step => step.name === CheckoutStepName.DELIVERY_OPTION,
+  )?.state
+
+  return (
+    <Flex flexDirection="column" backgroundColor="mono0">
+      <Box
+        p={2}
+        hidden={stepState !== CheckoutStepState.UPCOMING}
+        data-testid="DeliveryOptionsStep"
+      >
```

Comment:
Why make the box inside the flex hidden? I think the Stack component that wraps all steps [around here](https://github.com/artsy/force/pull/15675/files#diff-0216febf9295d715c239f647994c82f3555628d7b1abd08ad3c78e4ba9408e5aR82) will insert a gap unless this step returns null (could be wrong).